src/uct/ib/rc/accel/rc_mlx5_iface.c

```diff
@@ -214,7 +214,8 @@ uct_rc_mlx5_iface_handle_failure(uct_ib_iface_t *ib_iface, void *arg,
         txwq_copy.qend = UCS_PTR_BYTE_OFFSET(txwq_copy.qstart, txwq_size);
     }
 
-    if (uct_rc_mlx5_ep_handle_failure(ep, status) == UCS_OK) {
+    if ((status == UCS_ERR_ENDPOINT_TIMEOUT) &&
```

Comment:
IMO we need to call `uct_rc_mlx5_ep_handle_failure` for any status to `set_ep_failed` and notify user, the same for other similar places

Reply:
@evgeny-leksikov in other cases ucx will fail with Fatal error since it would be a real issue

Reply:
we already have a method to report error to user - error callback (if set), for debug purposes user can set `diag` log level